docs/control-structures.rst

```diff
@@ -67,12 +67,12 @@ Input parameters and output parameters can be used as expressions in
 the function body.  There, they are also usable in the left-hand side
 of assignment.
 
-...note::
+.. note::
     You cannot return some types from non-internal functions. If you enable the
     new experimental ``ABIEncoderV2`` feature by adding ``pragma experimental
     ABIEncoderV2;`` to your source file then more types are available, but
-    complex types (e.g., `mapping`) are limited to inside a single contract and
-    you cannot transfer them.
+    complex types (e.g., ``mapping``) are limited to inside a single contract
```

Comment:
mapping is the only complex type with this restriction.

Reply:
@chriseth Updated